07_git_exercises/danielreuven_ex1/README

```diff
@@ -0,0 +1,56 @@
+
+Git Basics
+1. command "echo 1 > abc.txt"
+2. command "git status", red
+3. command "git add abc.txt"
+   command "git status", green
+   command "git commit -m "added to index abc.txt" "
+   command "git status", changed to blue
+4. command "echo 2 >> abc.txt"
+5. command "git status", red
```

Comment:
The file color in PyCharm should be blue here, since there are changes in the working tree that were not committed yet